### PR TITLE
Add almalinux:8 support

### PR DESCRIPTION
**Pull request by alanking**

### CODE REVIEW

File: irods_core_builder.almalinux8.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  dnf clean all && \
+  rm -rf /var/cache/dnf /tmp/*
+
+# python3-devel must be installed because pyodbc requires building
+RUN dnf install -y \
+        gcc-c++ \
+        make \
+        python3-devel
+
+# For Python3 modules not available as packages in Centos 7:
```
Comment (trel) [inline irods_core_builder.almalinux8.Dockerfile:36]: mentions centos7 here...

File: irods_core_builder.almalinux8.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  #&& \
+  #yum clean all && \
+  #rm -rf /var/cache/yum /tmp/*
+
+# NOTE: This step cannot be combined with the installation step(s) above. Certain packages will
+# not be installed until certain other packages are installed. It's very sad and confusing.
+RUN \
+  dnf update -y && \
+  dnf install -y \
+    epel-release \
```
Comment (trel) [inline irods_core_builder.almalinux8.Dockerfile:61]: epel-release should be available ... at the beginning/immediately?

File: externals_builder.ubuntu20.Dockerfile [hunk 3]
```diff
@@ -27,6 +27,7 @@ RUN \
   hash -r
 
 ENV file_extension="deb"
+ENV package_manager "apt"
```
Comment (korydraughn) [inline externals_builder.ubuntu20.Dockerfile:30]: why do some `ENV` lines use the assignment operator while others don't?

Comment (alanking) [reply externals_builder.ubuntu20.Dockerfile:30]: Hmm.... a fine question. I will make these consistent.

File: irods_core_builder.almalinux8.Dockerfile (hunk 2 above)
Comment (alanking) [reply irods_core_builder.almalinux8.Dockerfile:61]: Indeed. I will see if it still works after removing here as it was installed at the very beginning, like you said.

File: irods_core_builder.almalinux8.Dockerfile (hunk 1 above)
Comment (alanking) [reply irods_core_builder.almalinux8.Dockerfile:36]: Will update

**Comment by alanking** [comment]
I accidentally squashed the almalinux-related changes into the original commit, hence the force-push :/

File: irods_core_builder.almalinux8.Dockerfile (hunk 2 above)
Comment (alanking) [reply irods_core_builder.almalinux8.Dockerfile]: We did it

**Pull request merged**